apps/src/lib/ui/lti/feedback/LtiFeedbackBanner.tsx

```diff
@@ -0,0 +1,171 @@
+   */
+  const [status, setStatus] = useState<string>(() => {
+    if (!currentUser.isLti || !currentUser.isTeacher) return UNAVAILABLE;
+
+    let status = tryGetLocalStorage(key, '');
+    if (status === UNAVAILABLE) status = '';
+
+    !status && fetchLtiFeedback(ltiFeedbackAction);
+
+    return status;
+  });
+
+  /**
+   * Effect for updating the local storage whenever the status changes.
+   */
+  useEffect(() => {
+    trySetLocalStorage(key, status);
+  }, [key, status]);
+
+  /**
+   * Effect for updating the status based on the LTI feedback.
+   */
+  useEffect(() => {
+    if (ltiFeedback === null) {
+      setStatus(AVAILABLE);
+    } else if (ltiFeedback) {
+      setStatus(ANSWERED);
+    }
+  }, [ltiFeedback]);
+
+  /**
+   * Effect for handling errors.
+   */
+  useEffect(() => {
+    error && setStatus('');
+  }, [error]);
+
+  /**
+   * Function for handling user feedback.
+   */
+  const answer = (satisfied: boolean) =>
+    createLtiFeedback(ltiFeedbackAction, {satisfied});
+
+  /**
+   * Function for closing the banner.
+   */
+  const close = () => setStatus(CLOSED);
+
+  return (
+    <Fade in={[AVAILABLE, ANSWERED].includes(status)} unmountOnExit={true}>
+      <Alert
+        key={key}
+        bsStyle="info"
+        className="lti-feedback-banner"
+        aria-labelledby="lti-feedback-banner-title"
+        closeLabel={i18n.closeDialog()}
+        onDismiss={status === ANSWERED ? close : undefined}
+      >
+        <FontAwesome
+          icon="hand-wave"
+          className="fa-fw"
+          title=""
+          aria-hidden="true"
+        />
+
+        <Fade in={!isLoading}>
+          {status === AVAILABLE ? (
+            <span>
+              <span id="lti-feedback-banner-title" aria-hidden="true">
+                {i18n.lti_feedbackBanner_question()}
+              </span>
+
+              <span className="lti-feedback">
+                <button
+                  type="button"
+                  title={i18n.lti_feedbackBanner_answer_positive()}
+                  onClick={() => answer(true)}
+                >
+                  <FontAwesome
+                    icon="thumbs-o-up"
+                    className="fa-fw"
+                    title=""
+                    aria-hidden="true"
+                  />
+                </button>
+
+                <button
+                  type="button"
+                  title={i18n.lti_feedbackBanner_answer_negative()}
+                  onClick={() => answer(false)}
+                >
+                  <FontAwesome
+                    icon="thumbs-o-down"
+                    className="fa-fw"
+                    title=""
+                    aria-hidden="true"
+                  />
+                </button>
+              </span>
+            </span>
+          ) : (
+            <span>
+              <span id="lti-feedback-banner-title" aria-hidden="true">
+                {i18n.lti_feedbackBanner_shareMore_text()}
+              </span>
```

Comment:
It's because we already use it as the alert title:

https://github.com/code-dot-org/code-dot-org/blob/f9fb91cc6df63eb7d0c780d96b1555f389214722/apps/src/lib/ui/lti/feedback/LtiFeedbackBanner.tsx#L101

<img width="319" alt="unasnwered" src="https://github.com/code-dot-org/code-dot-org/assets/11708250/06eac2c1-dfad-45f5-9f53-558ef4ccf00a">